blocks/editable/tinymce.js

```diff
@@ -2,45 +2,18 @@
  * External dependencies
  */
 import tinymce from 'tinymce';
-import { isEqual } from 'lodash';
 import classnames from 'classnames';
 
 /**
  * WordPress dependencies
  */
-import { Component, Children, createElement } from '@wordpress/element';
+import { Component, createElement } from '@wordpress/element';
 
 export default class TinyMCE extends Component {
 	componentDidMount() {
 		this.initialize();
 	}
 
-	shouldComponentUpdate() {
-		// We must prevent rerenders because TinyMCE will modify the DOM, thus
-		// breaking React's ability to reconcile changes.
-		//
-		// See: https://github.com/facebook/react/issues/6802
-		return false;
```

Comment:
I don't fully understand why this is being changed. I would also feel more comfortable if @aduth had a look at these parts. :)

Reply:
basically we were putting the defaultValue as a children of the TinyMce Node, so we couldn't allow preact to rerender this node, because its children can be mangled by TinyMCE.

With this new approach, it's simpler, we don't touch the children so we can allow React to rerender and update the attributes (styles, classnames,...) but it won't touch the children.

Reply:
Preact eh? 😄